### ISSUE TITLE
[Competition Bug]: UI Bug

### ISSUE THREAD

**Issue by Roovys** [bitsongofficial/sinfonia-ui#192]
### Prerequisites

- [X] You confirm that your bug is available through the Chrome browser accessed by a Desktop computer.
- [X] You read the guidelines for contributing.
- [x] You are able to reproduce the problem in the latest version of Sinfonia.
- [X] You checked that the problem has not yet been reported (also through closed issues).

### Describe the exact steps to reproduce the problem in as many details as possible

First issue, after I approve my transaction, a message of the transaction status should appear at the top of the website, similar to "transaction broadcasting."

However, if you go to dark/light mode while the notification is still active, the notification will appear strange.

Second issue: You can see the swap transaction status on the website when you are doing a transaction, but when the transaction fails, it does not show the right transaction status.

### Describe the issue behavior

As you can see in the first video, the notification did not match the website's theme, and when I switched to light mode, the alerts still in dark mode.

i think the notification colour should be following current website theme

https://user-images.githubusercontent.com/91830138/167460180-860d6a2c-2341-4bed-8de0-f0d30449126a.mp4

Second issue: As you can see in the second video, even if my transaction fails, the transaction history still shows "broadcasting," which I believe should be changed to "failed.

The unsuccessful transaction history still shows "broadcasting" even after I refreshed the website.

Edit 07:06
Tuesday, May 10, 2022
Coordinated Universal Time (UTC)
: I just realized the team already reported this "broadcasting" issues. but i think the first one is not reported yet.

https://user-images.githubusercontent.com/91830138/167461200-92cc48d1-7500-423a-8bfe-1f54ba0d6533.mp4

"

### BitSong Public Address

bitsong1fzgx606mmmhv0qwwgp32u30j24vz0yv9l8fcg8

### Osmosis Public Address

osmo1fqgg5pvzma2znu9w5rhglshjwc2nlzn878q0c5

### Which browser are you using?

Google Chrome

### Which is your browser version?

 Google Chrome	101.0.4951.54 (Official Build) (64-bit) (cohort: Stable)

### Which kind of device are you using?

Desktop

### Are you using a ledger?

No

### Agree the Competition Rules

- [X] You agree the competition rules

**Comment by Giorgionocera** [comment]
The first issue is a duplicate of one issue shown in #180. The second is a duplicate of #172.

**Comment by Roovys** [comment]
> The first issue is a duplicate of one issue shown in #180. The second is a duplicate of #172.

Yes. But #180 ,i think he edited the submission, so its like he the first one who reported this notification bug. 

U can see his edit history, he added this notification bug.

**Comment by Giorgionocera** [comment]
You are right. I'm sorry for the mistake on the first issue.

**Comment by Giorgionocera** [comment]
Hi there @Roovys and many thanks for the time spent on Sinfonia during the competition.

Many thanks also for the issues you reported. However, after a careful analysis of our technical jury, the issues you found haven’t been considered as major or critical thus meaning that we couldn’t assign you the minimum incentive for them.

However, the team appreciated the effort and decided to incentivize you with 1500 BTSG for the submitted report.

That having been said, we thank you again for your time and effort and we are looking forward to sending you the incentive in the BitSong wallet you submitted with the report. So please confirm that the following wallet address is correct:

```bitsong1fzgx606mmmhv0qwwgp32u30j24vz0yv9l8fcg8```

Kind Regards

**Comment by cbsantos90** [comment]
lol. i didn't even know this post existed
